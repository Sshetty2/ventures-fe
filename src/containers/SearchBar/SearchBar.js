import React, { useState } from 'react';
import searchIcon from '../../images/search.svg';

export const SearchBar = () => {
  const [searchQuery, updateSearchQuery] = useState('');

  return (
    <div className='SearchBar'>
      <h3 className='searchbar-title'>SEARCH EVENTS</h3>
      <div className='searchbar-container'>
        <img src={searchIcon} alt='search icon' className='searchbar-icon' />
        <input onChange={(e) => updateSearchQuery(e.target.value)} name='searchQuery' placeholder='Search all events' className='searchbar-input' value={searchQuery}/>
      </div>
    </div>
  );
}

export default SearchBar;
