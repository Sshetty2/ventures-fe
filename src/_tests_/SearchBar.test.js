import React from 'react';
import { shallow } from 'enzyme';
import { SearchBar } from '../containers/SearchBar/SearchBar';

describe('SearchBar', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SearchBar />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});