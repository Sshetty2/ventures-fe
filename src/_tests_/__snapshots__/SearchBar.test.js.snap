// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should match the snapshot 1`] = `
<div
  className="SearchBar"
>
  <h3
    className="searchbar-title"
  >
    SEARCH EVENTS
  </h3>
  <div
    className="searchbar-container"
  >
    <img
      alt="search icon"
      className="searchbar-icon"
      src="search.svg"
    />
    <input
      className="searchbar-input"
      name="searchQuery"
      onChange={[Function]}
      placeholder="Search all events"
    />
  </div>
</div>
`;
